Documentation/spelling_wordlist.txt

```diff
@@ -49,6 +49,7 @@ Donenfeld
 Elasticsearch
 FIt
 Facebook
+Failover
```

Comment:
This one is not needed, the lowcase version below should make the exception case-insensitive.